doc/source/installation.rst

```diff
@@ -160,6 +161,12 @@ After installing pandoc, install sphinx on your device.
 
     sudo apt install python3-sphinx
 
+Install IPython
+
+.. code-block:: shell
+
+    sudo pip3 install ipython
+
```

Comment:
Why is ipython relevant for building the documentation ?

Reply:
Ipython is irrelevant for pycram.

Reply:
@artnie Could you remove the codeblock from the documentation

Reply:
Ah I see, it's just warnings you get from building the ipynb files when ipython is not installed.

Reply:
with building the ipynb files you mean building the doc?